build_modules/README.md

```diff
@@ -32,3 +32,70 @@ building your Dart code.
 
 [`build_web_compilers`]: https://pub.dartlang.org/packages/build_web_compilers
 [`build_vm_compilers`]: https://pub.dartlang.org/packages/build_vm_compilers
+
+## Module algorithms
```

Comment:
I think this should be moved to right above the actual module algorithms, and this title should be renamed `Module discovery strategy`, `Module creation`, or something like that?

Reply:
Done